airflow/api_connexion/openapi/v1.yaml

```diff
@@ -4810,6 +4816,56 @@ components:
       required: true
       description: The task ID.
 
+    EventLogDAGID:
+      in: query
+      name: dag_id
+      schema:
+        type: string
+      required: false
+      description: DAG ID of an event log.
```

Comment:
We have already a dag id param: https://github.com/apache/airflow/blob/6c649aefd2dccbc1765c077c5154c7edf384caeb/airflow/api_connexion/openapi/v1.yaml#L4797-L4803

so I don't know if we should use it or create a new one, @pierrejeambrun wdyt?

Reply:
I think we should re-use it if this is exatly the same thing. No need to duplicate I believe

Reply:
Actually this `dag_id` parameter works for `path` and not as query parameter. In addition, it's set as `required` while `eventlogs`'s filtering parameters aren't strictly required.

Reply:
In this case, maybe we can prefix that with `Filter` and put them in the Filter section where other filters reside in the spec. Also maybe a generic name such as `FilterDAGID` so we can easily reuse it. (shouldn't be specific to EventLog I believe)

Reply:
Parameters are now converted to filters in latest commit. Thanks!